### ISSUE TITLE
Does not support multiple arguments

### ISSUE THREAD

**Issue by dzuelke** [npm/node-which#19]
Given multiple arguments, `which` is supposed to print the location of each match, and return the number of errors as the exit code.

For example:

```
$ which ls mkdir
/bin/ls
/bin/mkdir
$ echo $?
0
$ which doesnotexist mkdir
/bin/mkdir
$ echo $?
1
```

However, this package doesn't behave that way:

```
$ node_modules/.bin/which ls mkdir
/bin/ls
$ echo $?
0
$ node_modules/.bin/which doesnotexist mkdir
not found: doesnotexist
$ echo $?
127
```

**Comment by dzuelke** [comment]
(and then with `.node_modules/bin/` on `$PATH`, Bash stuff that uses `which` just magically blows up)

**Comment by isaacs** [comment]
Patch welcome to make it behave identical to `which(1)`!

**Comment by dzuelke** [comment]
Fine... see #20 !

**Comment by isaacs** [comment]
Landed on 1.2.0
